### PR TITLE
feat: generate token and create own token

### PR DESCRIPTION
**Pull request by BeierKevin**
# PR description
*Describe your changes in detail here*

Now you can regenerate a cryptographic token or you are able to create your own token by writing in the text field.
Currently, the rules are very simple, they have to be specified by the team in a more consice way. Additionally, I set it up so that the notification system is incorporated in this.

# Definition Of Done (DoD)
*This PR can be squashed / merged if*
- [x] a developer is assigned
- [x] the PR is NOT estimated
- [x] the PR is labeled
- [x] the PR is NOT assigned to the current sprint
- [x] a meaningful title has been set according to https://www.conventionalcommits.org/
- [x] the PR is described in detail
- [x] the PR links to an issue
- [x] the PR has been reviewed

*Add additional conditions here if necessary for this PR*

Fix: #118

### CODE REVIEW

**Comment by BeierKevin** [comment]
The current rules for the token are just placeholders because I am unsure what the rules should be therefore we need to decide what is important

**Comment by Claiyc** [comment]
> The current rules for the token are just placeholders because I am unsure what the rules should be therefore we need to decide what is important
> 
>     1. min characters
> 
>     2. what characters have to be in
> 
>     3. numbers ?
> 
>     4. etc.

see #118 reply

**Comment by BeierKevin** [comment]
I ran into something very strange but i don't know why

with this updated function so every generated token has all the properties
image


I reach the maximum call stack size

image

**Review by Claiyc — CHANGES_REQUESTED** [review]
session stopped message should not appear when session is not running

File: src/App.vue [hunk 1]
```diff
@@ -412,7 +412,7 @@ function validateAccessToken() {
     isAccessTokenValid.value = isValid;
 
     if (!isValid) {
-        stopSession();
+        if (isSessionActive.value) stopSession();
```
Comment (Claiyc) [inline src/App.vue:415]: no. Just no

**Review by Claiyc — APPROVED** [review]
lgtm

**Pull request merged**